### DISCUSSION TITLE
Deployment Updates

### DISCUSSION THREAD

**Discussion by tiprock-network** [tiprock-network/MedicineDemand-TimeSeries#10] [Announcements]
Good work @aaryan2134 I am also done with my deployment. We are now at this point using these technologies: Power Apps, Azure ML and Azure Web Service. To merge all the apps we have created we are going to use iframes to make on full application.

@varma59 I will also integrate the bot we have and the final product will be Heath Box.
Kindly check the main repository. It has been updated.

**Comment by aaryan2134** [comment]
Great! Sounds awesome

**Comment by tiprock-network** [comment]
The UI is now officially deployed with my model being integrated. In order to integrate your app @aaryan2134  and @varma59 kindly provide me the url links so I can create iframes.

**Reply by aaryan2134** [reply]
https://apps.powerapps.com/play/e/cf46801e-aa6d-4a32-a161-692e900c34cc/a/6f7695e5-11d7-4fa1-96e9-bd0aae6bd6f7?tenantId=84c31ca0-ac3b-4eae-ad11-519d80233e6f

Yes here is the app link
You can add this as an iframe or keep it as a link as well

**Reply by tiprock-network** [reply]
I have accessed the link although I need a subscription. Is there a way you can share or even save the actual app and serve it for public use.
I find that ease.

**Reply by aaryan2134** [reply]
Actually that is not possible. We need a subscription to access the app as it uses an API and premium features

**Reply by aaryan2134** [reply]
So, no worries, I can show it during the demo as I have an active subscription
